main/mainworker.h

```diff
@@ -40,6 +40,7 @@ class MainWorker
 	int FindDomoticzHardware(int HwdId);
 	int FindDomoticzHardwareByType(const _eHardwareTypes HWType);
 	CDomoticzHardwareBase* GetHardware(int HwdId);
+	CDomoticzHardwareBase* GetHardwareBase(const std::string &HwId, const int Type);
```

Comment:
Can you rename the function/parameters please, this is more consistent with what we have

CDomoticzHardwareBase* GetHardwareByIDType(const int HwdId, const _eHardwareTypes HWType);